Formula/libxml++4.rb

```diff
@@ -0,0 +1,46 @@
+class Libxmlxx4 < Formula
+  desc "C++ wrapper for libxml"
+  homepage "https://libxmlplusplus.sourceforge.io/"
+  url "https://download.gnome.org/sources/libxml++/4.0/libxml++-4.0.0.tar.xz"
+  sha256 "4f26b5fdb9ebd91e440d60343ac82400f88287facedc7e81b95f23d002f8049f"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "glibmm"
+
+  uses_from_macos "libxml2"
+
+  def install
+    ENV.cxx11
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <libxml++/libxml++.h>
+
+      int main(int argc, char *argv[])
+      {
+         xmlpp::Document document;
+         document.set_internal_subset("homebrew", "", "https://www.brew.sh/xml/test.dtd");
+         xmlpp::Element *rootnode = document.create_root_node("homebrew");
+         return 0;
+      }
+    EOS
+    ENV.libxml2
```

Comment:
I don't think this exists

Reply:
That's fine, this example is just constructing a simple XML tree, it is not actually trying to fetch the url.

Reply:
? I mean the ENV statement, I don't think it does anything

Reply:
Aha, I removed that line, seems unnecessary on my local machine.